packages/playground/blueprints/src/lib/steps/login.spec.ts

```diff
@@ -27,7 +28,30 @@ describe('Blueprint step login', () => {
 	it('should log the user in', async () => {
 		await login(php, {});
 		const loginResponse = await handler.request({
-			url: '/wp-login.php',
+			url: '/wp-admin/',
+		});
+		expect(loginResponse.httpStatusCode).toBe(302);
+		expect(loginResponse.headers['location']).toHaveLength(1);
+		const initialRedirectUrl = new URL(
+			loginResponse.headers['location'][0]
+		);
+		expect(initialRedirectUrl.pathname).toBe('/wp-admin/');
+
+		const adminResponse = await handler.request({
+			url: '/wp-admin/',
+		});
+		expect(adminResponse.httpStatusCode).toBe(200);
+		expect(adminResponse.text).toContain('Dashboard');
+	});
+
+	it('should log the user in if the auto login parameter is set', async () => {
```

Comment:
Nit, let's make this name mire precise. When this fails, we'll see the test name in red in CLI. Let's be kind to our future selves and communicate this is about the query string parameter, say:

should log the user in if the `playground_force_auto_login_as_user ` query parameter is set